caf: ble_state: Remove TX power update using HCI command on connected (#12234)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    * The library by improving broadcast of :c:struct:`module_state_event`.
      The event informing about entering either :c:enum:`MODULE_STATE_READY` or :c:enum:`MODULE_STATE_OFF` is not submitted until the CAF Bluetooth LE advertising module is initialized and ready.

* :ref:`caf_ble_state`:

>    * Removed TX power update using a Bluetooth HCI command for SoftDevice Blutooth LE Link Layer (:kconfig:option:`CONFIG_BT_LL_SOFTDEVICE`) right after connection is established.

Reviewer: ```suggestion
   * Removed TX power update using a Bluetooth HCI command for SoftDevice Bluetooth LE Link Layer (:kconfig:option:`CONFIG_BT_LL_SOFTDEVICE`) right after a connection has been established.
```

>      The :kconfig:option:`CONFIG_BT_CTLR_TX_PWR` Kconfig choice can be used to set the TX power used for both advertising and connections also for the SoftDevice Link Layer.

Reviewer: ```suggestion
     The :kconfig:option:`CONFIG_BT_CTLR_TX_PWR` Kconfig option can be used to set the TX power for advertising and connections also for the SoftDevice Link Layer.
```

Reviewer 2: Also, we should have a space between line no: 792 and 793 to render correctly in the docs.
